app/tsconfig.json

```diff
@@ -74,10 +74,8 @@
     "node_modules",
     "dist",
     "postcss.config.js",
-    "vendor",
+    "vendor/ace-builds",
```

Comment:
This should not be necessary. We no longer use the Ace Editor.

Reply:
Absolutely :)